cli/config/config.go

```diff
@@ -91,12 +91,6 @@ func GetComposeFile(isProfile bool, composeName string) (string, error) {
 		return composeFilePath, nil
 	}
 
-	log.WithFields(log.Fields{
```

Comment:
Are we sure we want to remove this trace log? I sent a PR #946 precisely adding them back when extracting the packed compose directory.

Reply:
This log doesn't really make sense anymore because everything that would've been extracted would have happened already.

Reply:
We could rework the log to not say extracting anymore? That way we can trace if it finds a docker-compose or not. wdyt?

Reply:
Let's do it as you suggest

Reply:
@mdelapenya Ok pushed a new commit, let me know what you think

Reply:
Please take a look at #946 to be merged to this branch